### ISSUE TITLE
Some message-compose dialogs go full-screen

### ISSUE THREAD

**Issue by Smiles2go** [hape42/dailygammon#125]
Version 2.0 (674) from 22. June 2024 07:24:42

I like the new messaging approach, thank you. Not sure why some message-compose situations eat whole screen while most use the smaller window where we can see relevant information like the board position. Why do I get this?

![image](https://github.com/hape42/dailygammon/assets/121371957/294a51ba-46d1-44d2-9ac8-ef834f7b589d)

Generally prefer this:
![image](https://github.com/hape42/dailygammon/assets/121371957/899a3f17-ef39-4394-9757-6ebf05d5be30)

**Comment by hape42** [comment]
When a message like "smiles2go wins 2 points and the match." appears, dg no longer displays a board. Even if you play in the browser, only this information is displayed on a mostly blank screen.

**Comment by Smiles2go** [comment]
I see the situations are different, so seems fine for now. 

For a more consistent UI, I might still expect to see the same sort of message-compose view anyway (might even be a bit simpler to code).

**Comment by hape42** [comment]
Since the server has no api, I have to read every single HTML page. I only have the information that is on this read page.  I have tried to derive more from the information. But that only made things worse.

The only way that made sense to me is to parse HTML page by HTML page and only display what I find on that page in IOS.

This may seem inconsistent at times, but that's the way Dailygammon provides it.

I also sometimes forget how Dailygammon works in the browser. Then I play a few turns in the browser and then I realise again why I solved some things this way and that.

Without my own game logic, I am bound to DailyGammon processes.
